src/tools/compiletest/src/runtest/coverage.rs

```diff
@@ -18,7 +18,7 @@ impl<'test> TestCx<'test> {
             .unwrap_or_else(|| self.fatal("missing --coverage-dump"))
     }
 
-    pub(crate) fn run_coverage_map_test(&self) {
+    pub(super) fn run_coverage_map_test(&self) {
```

Comment:
Remark: changed visibility from `pub(crate)` -> `pub(super)` for consistency as it does not need to be visible outside of `runtest.rs`.